tests/functionnal/NotificationTargetTicket.php

```diff
@@ -306,13 +308,15 @@ public function testTimelineTag() {
 
       $basic_options = [
          'additionnaloption' => [
-            'usertype' => ''
+            'usertype' => '1',
+            'is_self_service' => true,
```

Comment:
Same remarks here.

Reply:
```usertype``` is only used for ```noAuto``` but without this key we have a ```NOTICE```

```Error E_NOTICE in /home/dev/GLPI/master/tests/functionnal/NotificationTargetTicket.php on line 318, generated by file /home/dev/GLPI/master/inc/notificationtargetticket.class.php on line 160:```

```is_self_service``` is a notification option (but computes internally) and need to be set manually because we call ```getDataForObject``` directly rather than using the entire process from the ```Raisevent````